client/blocks/domain-to-plan-nudge/index.jsx

```diff
@@ -299,7 +299,15 @@ class DomainToPlanNudge extends Component {
 							}
 						</Button>
 						<div className="domain-to-plan-nudge__credit-card-info">
-							{ translate( 'Using credit card ****%s', { args: storedCard ? storedCard.card : 'xxxx' } ) }
+							<a
```

Comment:
Hmm. While this doesn't pollute the upgrade nudge with additional button, I think it's not very UX friendly. I doubt anybody would know that by clicking on that they can change CC.

Maybe we could at least add a small text like "Using credit card ****1234 (click to change)" or "Using credit card ****1234 (change)". Still not the best solution, but I don't have anything better atm. ;)